### PR TITLE
Update XEP-0298

### PR DESCRIPTION
**Pull request by saghul**
Ported original changes from https://github.com/emcho/coin

### CODE REVIEW

**Comment by saghul** [comment]
Here it goes, @stpeter! Also, hi @emcho!

**Comment by linuxwolf** [comment]
This PR is on hold until we get confirmation from the existing authors about the changes.  Once received, we'll get the PR applied.

File: xep-0298.xml [hunk 1]
```diff
@@ -30,22 +30,22 @@
...
+    </author>
+    <author>
+      <firstname>Saúl Ibarra</firstname>
+      <surname>Corretgé</surname>
+      <email>saul@ag-projects.com</email>
+      <jid>saul@ag-projects.com</jid>
     </author>
     <discuss>jingle</discuss>
     <revision>
-      <version>0.1</version>
```
Comment (linuxwolf) [inline xep-0298.xml]: The existing revision cannot be removed; please add it back.

Also, it seems necessary to add a <revision> for 0.2.  Does the following look acceptable?

    <revision>
      <version>0.2</version>
      <date>2015-07-02</date>
      <initials>sic</initials>
      <remark><p>Correcting errors in grammar and examples; aligning closer to dependent specifications.</p></remark>
    </revision>

Comment (saghul) [inline xep-0298.xml]: It does! I'll update the PR.

**Comment by saghul** [comment]
@linuxwolf force-pushed addressing your suggestions.

**Comment by stpeter** [comment]
I talked with @emcho and he said he's fine with the contribution. If he doesn't say something here soon I'll poke him again. :-)

**Comment by emcho** [comment]
I confirm! Thanks very much for doing the work guys!

**Comment by linuxwolf** [comment]
Thanks all. We'll get this published soon...ish (-:

- m&m
{mobile}

**Comment by linuxwolf** [comment]
Merged in commit 4b3918acace0581abf89254f0bddcebed3b7bc33

**Pull request closed without merging**